### PR TITLE
[GEN-1612] Use 16.7-consortium and v55 of BPC REDCap to cbio mapping table for CRC 3.1

### PR DESCRIPTION
**Pull request by rxu17**
**Purpose:** 

- Updates `_MG_RELEASE_SYNID` to use 16.7-consortium for CRC 3.1 release.
- Updating `_REDCAP_TO_CBIOMAPPING_SYNID`  to use version [55](https://www.synapse.org/Synapse:syn25712693.55) of the BPC REDCap to cbio mapping table. This version of the table contains:
   - newest updates to mapping table by cbioportal for crc 3.1
   - Set `CRC` column to `True` for where s`ampleType = TIMELINE-PERFORMANCE` and `data_type = portal_value` to support the export of the [data_timeline_performance_status.txt](https://www.synapse.org/Synapse:syn64322183) file

**Testing:**

- [X] Run sample run on staging and make sure it runs through. Do comparison report on results with previous.

Depends on #143

### CODE REVIEW

File: geniesp/bpc_redcap_export_mapping.py [hunk 1]
```diff
@@ -532,9 +533,11 @@ class BpcProjectRunner(metaclass=ABCMeta):
         "production":"syn21446571",
         "staging": "syn64018293"
     }
-    # main GENIE release folder (15.4-consortium)
+    # main GENIE release folder
     # Must use consortium release, because SEQ_YEAR is used
-    _MG_RELEASE_SYNID = "syn53170398"
+    # NOTE: Must match release tracking sheet and release table info
```
Comment (thomasyu888) [inline geniesp/bpc_redcap_export_mapping.py:538]: Should we link out to the release tracking sheet and table info here?

File: geniesp/bpc_redcap_export_mapping.py [hunk 2]
```diff
@@ -521,6 +521,7 @@ class BpcProjectRunner(metaclass=ABCMeta):
     _SPONSORED_PROJECT = ""
     # Redcap codes to cbioportal mapping synid and form key is in
     # version 38, 42 were last stable version(s)
+    # NOTE: Should be pointed towards latest version of table
```
Comment (thomasyu888) [inline geniesp/bpc_redcap_export_mapping.py:524]: There will be a new mapping version, did we want to wait until that is in to update both at the same time?

Comment (rxu17) [reply geniesp/bpc_redcap_export_mapping.py:524]: Yes this PR can be in place here.

File: geniesp/bpc_redcap_export_mapping.py (hunk 1 above)
Comment (rxu17) [reply geniesp/bpc_redcap_export_mapping.py:538]: I think it's already in the [SOP](https://sagebionetworks.jira.com/wiki/spaces/APGD/pages/2621931648/GENIE+BPC+Pipeline+SOP#4.-Processing%3A-update-references) so should be good on this front

File: geniesp/bpc_redcap_export_mapping.py (hunk 2 above)
Comment (danlu1) [reply geniesp/bpc_redcap_export_mapping.py:524]: Are we waiting on MSK cBioPortal team to share the newer mapping file?

Comment (rxu17) [reply geniesp/bpc_redcap_export_mapping.py:524]: Yes

**Review by Chelsea-Na — APPROVED** [review]

**Pull request merged**